scripts/install.sh

```diff
@@ -1,4 +1,18 @@
 #!/usr/bin/env bash
+#
+# This install script is used by the tutorial in the AWS docs tutorial on using
+# App Mesh with Kubernetes:
+#
+# https://docs.aws.amazon.com/eks/latest/userguide/mesh-k8s-integration.html
+#
+# Do not modify it without testing the steps from the tutorial.  The default
+# branch and version below will be installed by users following the tutorial,
+# so the version should be updated with each release.
+#
+
+REPO=${REPO:-aws/aws-app-mesh-inject}
+BRANCH=${BRANCH:-master}
+VERSION=${VERSION:-v0.1.6}
```

Comment:
Ah, so [`ca-bundle.sh`](https://github.com/aws/aws-app-mesh-inject/blob/master/scripts/ca-bundle.sh#L15-L18) will inherit `VERSION` and expand it when [line 24](https://github.com/aws/aws-app-mesh-inject/blob/master/scripts/deployInjector.sh#L24) in `deployInjector.sh` is executed.

Probably best to check that `VERSION` is set in `deployInjector.sh` as well, at least if `IMAGE_NAME` is not set. I assume you don't want to set the default in more than one place, but I think we should verify it has a value in `deployInjector.sh`, or we might apply a broken Image path to the cluster if someone just runs it directly.

Reply:
done.